distance/gen_distance_orig.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-set -euo pipefail
-
```

Comment:
Re-enable to fail fast?

Reply:
When I used AFLGo in the first time, most of these `*.sh` scripts unexceptionally got stuck or exited with weird and confusing behaviors (finally I understand and solved these issues after further reading the code and docs).

As you can see, in 769f5ac I added `set -euo pipefail` for each of them which ensures the script could immediately exit if any command (also those in the pipeline) has a non-zero exit status or uses undefined variable, so users can avoid stepping in worse unexpected behaviors.

However, later I found that `gen_distance_orig.sh` has its own error handling processes for pipelines such as 
https://github.com/aflgo/aflgo/blob/ac9246a4fe7c0bba15e36c994c456af14d89b698/scripts/genDistance.sh#L98-L105 And `set -euo pipefail` would just break it and make the script silently exit.

So in 1e9c90c I removed it and explained the reason.